MapboxNavigation/NavigationViewController.swift

```diff
@@ -200,6 +200,26 @@ open class NavigationViewController: UIViewController, NavigationStatusPresenter
     
     var bottomViewController: ContainerViewController?
     
+    public typealias FloatingButtonsPositions = NavigationView.FloatingButtonsPositions
+    
+    open var floatingButtonsPosition: FloatingButtonsPositions? {
+        get {
+            return mapViewController?.floatingButtonsPosition
+        }
+        set {
+            mapViewController?.floatingButtonsPosition = newValue
+        }
+    }
+    
+    open var floatingButtons: [UIButton]? {
```

Comment:
Should we require these buttons to be of type FloatingButton so that custom buttons will look and behave consistently? If so, FloatingButton and `FloatingButton.rounded(image:selectedImage:size:)` should be documented and made public.

Reply:
Thanks for the suggestion. I'll keep it to be `UIButton` to be more versatile in development, but add the reference of the   `FloatingButton.rounded(image:selectedImage:size:)` in the documentation.